git/helpers.sh

```diff
@@ -0,0 +1,59 @@
+#
+# Deletes a branch locally and remotely
+#
+function gitdelete() {
+  if [ -n "$1" ]; then
+    local_exists=`git show-ref refs/heads/$1`
+    remote_exists=`git ls-remote origin $1`
+    if [ -n "$local_exists" ]; then
+      git branch -D $1;
+    else
+      echo "could not find local branch: $1";
```

Comment:
should we add `exit`?

Reply:
The function was designed to delete the branch locally and remotely; an `exit` here defies the purpose.

Reply:
Ah yes; wasn't thinking